public/scripts/d3/margins.js

```diff
@@ -17,6 +17,8 @@ var echo = {};
     .append("g")
       .attr("transform", "translate(0,0)");
 
+  self.jsonUrl = 'https://gist.githubusercontent.com/ConchobarMacNessa/8d852e941cf9c86c98b4b22a269f462e/raw/243656f4ea084136e16851767206bda6932e78d1/EchoTestData.json'
```

Comment:
The reason we have our data in a gist is because we can't access local files in the browser with d3 apparently. The work around I found online was just to host the data in a gist, which I thought would be fine when we're just using dummy data.